llvm/lib/Target/RISCV/RISCVTargetMachine.cpp

```diff
@@ -95,11 +95,6 @@ static cl::opt<bool>
                         cl::desc("Enable Split RegisterAlloc for RVV"),
                         cl::init(true));
 
-static cl::opt<bool> EnableMISchedLoadClustering(
-    "riscv-misched-load-clustering", cl::Hidden,
-    cl::desc("Enable load clustering in the machine scheduler"),
-    cl::init(false));
```

Comment:
In this patch, please just flip the bool default here.  If you want to remove the option entirely, do that in a separate change in a couple weeks.